frontend/build.rs

```diff
@@ -0,0 +1,107 @@
+///
+/// Build Script
+/// This is run as a pre-build step -- before the rust backend is compiled.
+/// NOTE: Should be included in root's build script
+///
+use std::ffi::OsStr;
+use std::fmt::Display;
+use std::fs::File;
+use std::path::{Path, PathBuf};
+use std::{fs, io};
+use std::{io::Write, process::Command};
+
+#[cfg(target_family = "windows")]
+const SHELL: &str = "cmd";
+#[cfg(target_family = "unix")]
+const SHELL: &str = "sh";
+#[cfg(target_family = "windows")]
+const SHELL_ARG: &str = "/C";
+#[cfg(target_family = "unix")]
+const SHELL_ARG: &str = "-c";
+
+/*
+ * Note 1: this file was written for *nix systems -- it likely won't
+ * work on windows!
+ */
+
+#[allow(dead_code)]
+fn shell(command: impl AsRef<OsStr> + Display) {
+    // println!("build.rs => {}", command);
+
+    let output = Command::new(SHELL)
+        .arg(SHELL_ARG)
+        .arg(&command)
+        .output()
```

Comment:
You should verify that exit code is `0`. If it is not, then error should be generated (probably with `panic!`). 

I start build script and yarn returned error, but build continued and failed later during file copy